tests/rptest/util.py

```diff
@@ -375,7 +375,8 @@ def __enter__(self):
         """Isolate certain ips from the nodes using firewall rules"""
         cmd = [
             f"iptables -A INPUT -p tcp --{self.mode_for_input} {self._port} -j DROP",
-            f"iptables -A OUTPUT -p tcp --dport {self._port} -j DROP"
+            f"iptables -A OUTPUT -p tcp --dport {self._port} -j DROP",
+            f"ss -K dport {self._port}",
```

Comment:
nit: could you elaborate in the commit message what the problem looked like before this change?